dd-sdk-android/src/test/kotlin/com/datadog/android/DatadogInterceptorTest.kt

```diff
@@ -112,6 +142,12 @@ internal class DatadogInterceptorTest : TracingInterceptorNotSendingSpanTest() {
                 expectedStopAttrs
             )
         }
+
+        verify(mockRumAttributesProvider, times(1)).onProvideAttributes(
```

Comment:
not sure you need to test this here. You are already asserting the attributes above.

Reply:
agree, probably will remove these assertions as long as attributes are verified